src/static/js/publish_options.js

```diff
@@ -179,6 +179,15 @@ $(function() {
                                 '<option value="1">Do not generate and upload (not recommended)</option>', 
                                 classes: 'pbool preset' }
                         ]}, 
+                    { id: "tab-avidus", title: "AVID US Microchips", classes: 'localeus', 
```

Comment:
there should be a tag class for hasavid or something here so that it can be hidden on non-sheltermanager installations